docs/topics/application-design.md

```diff
@@ -173,6 +173,6 @@ that reaches the SIA acceptance or production environments had a succesfull run
 through the aforementioned checks.
 
 ### Logging
-SIA uses Sentry to track application errors. SIA HTTP traffic is logged in the
+SIA uses Azure insight monitoring to track application errors. SIA HTTP traffic is logged in the
```

Comment:
This should be discussed with the VNG more broadly than just in a PR.

Reply:
See PR https://github.com/Amsterdam/signals/pull/1091 Application Insights can now be enabled or disabled separately